src/MyEventEmitter.js

```diff
@@ -1,12 +1,98 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+const http = require('http');
+const server = new http.Server();
+
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (eventName in this.events) {
+      this.events[eventName].push(listener);
+    } else {
+      this.events[eventName] = [listener];
+    }
+
+    return this;
```

Comment:
- can be less
- don't see any sense for return `this` here
check other places 
```suggestion
    if (!this. events[eventName]) {
      this. events[eventName] = [];
    }
    this.events[eventName].push(listener);
```